webservices/resources/legal.py

```diff
@@ -160,15 +160,16 @@ def generic_query_builder(q, type_, from_hit, hits_returned, **kwargs):
         Search()
         .using(es_client)
         .query(Q("bool", must=must_query))
-        .highlight(
-            "text", "name", "no", "summary", "documents.text", "documents.description"
-        )
         .highlight_options(require_field_match=False)
         .source(exclude=["text", "documents.text", "sort1", "sort2"])
         .extra(size=hits_returned, from_=from_hit)
         .index(SEARCH_ALIAS)
         .sort("sort1", "sort2")
     )
+    if type_ == "advisory_opinions":
```

Comment:
I removed name and no from highlight because I noticed that even with name and no removed from keyword search, it was still highlighting name.

![image](https://github.com/user-attachments/assets/bbe9f851-e971-405a-b799-7498c936a2fc)